definitions/artifacts/azure-databricks-workspace.json

```diff
@@ -0,0 +1,87 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema",
+	"$id": "https://schemas.massdriver.cloud/definitions/artifacts/azure-databricks-workspace.json",
+	"$md": {
+		"access": "public"
+	},
+	"type": "object",
+	"title": "Azure Databricks Workspace",
+	"description": "",
+	"additionalProperties": false,
+	"required": [
+		"data",
+		"specs"
+	],
+	"properties": {
+		"data": {
+			"title": "Artifact Data",
+			"type": "object",
+			"required": [
+				"infrastructure",
+				"security"
+			],
+			"properties": {
+				"infrastructure": {
+					"title": "Infrastructure configuration",
+					"type": "object",
+					"required": [
+						"ari",
+						"url",
+						"raw_container_ari",
+						"scratch_container_ari",
+						"enriched_container_ari"
+					],
+					"properties": {
+						"ari": {
+							"$ref": "../types/azure-resource-id.json"
+						},
+						"url": {
+							"title": "Azure Databricks workspace URL",
+							"$ref": "../types/https-url.json"
+						},
+						"raw_container_ari": {
+							"title": "Raw Container",
+							"description": "The name of the container where the raw data will be stored.",
+							"$ref": "../types/azure-resource-id.json"
+						},
+						"scratch_container_ari": {
+							"title": "Scratch Container",
+							"description": "The name of the container where intermediate data transformations or experimental analyst data will be stored.",
+							"$ref": "../types/azure-resource-id.json"
+						},
+						"enriched_container_ari": {
+							"title": "Enriched Container",
+							"description": "The name of the container where curated data assets will be stored.",
```

Comment:
Should the descriptions be describing "The **resource ID** of the container" instead of "The **name** of the container"?

Reply:
good suggestion, done!